docs/userguide/plugins.rst

```diff
@@ -34,10 +34,9 @@ add on any wrappers that are needed to launch the command (eg srun inside
 slurm). Providers and launchers are usually paired together for a particular
 system type.
 
-A `Channel` allows the commands used to interact with an `ExecutionProvider` to be
-executed on a remote system. The default channel executes commands on the
-local system, but a few variants of an `parsl.channels.SSHChannel` are provided.
-
+Parsl also has a deprecated ``Channel`` abstraction.  See
+`issue 3515 <https://github.com/Parsl/parsl/issues/3515>`_
+for further discussion.
```

Comment:
I'm wondering if it would be smarter to go all the way and just remove the references in the documentation entirely.  That is, nothing will have changed functionally, so if folks still want to use it, they can reference an older document in the interim while complaining to us that we're removing a crucial element.  Meanwhile, under the expectation that ~no one _is_ using it, this saves work in the future&nbsp;&mdash;&nbsp;do the work once and completely.  If removing it from the title of this section, this paragraph becomes a bit of a non-sequitur.

Reply:
mostly I was erring on the side of putting in links to the issue in lots of places because of previous people popping up and saying "but i like channels".

it's possible that actually those people don't exist any more - but in that case, we could be deleting channels entirely right now.